rerun_bridge/src/rerun_bridge/rerun_ros_interface.cpp

```diff
@@ -332,6 +331,14 @@ void log_point_cloud2(
                 return;
             }
             has_z = true;
+        } else if (field.name == options.colormap_field.value_or("rgb")) {
+            rgb_offset = field.offset;
+            if (field.datatype != sensor_msgs::msg::PointField::UINT32 &&
+                field.datatype != sensor_msgs::msg::PointField::FLOAT32) {
+                rec.log(entity_path, rerun::TextLog("Only UINT32 and FLOAT32 rgb field supported"));
+                return;
+            }
+            has_rgb = true;
```

Comment:
Should we really `return`? can we just ignore the field instead (`continue`)?

```suggestion
            if (field.datatype == sensor_msgs::msg::PointField::UINT32 ||
                field.datatype == sensor_msgs::msg::PointField::FLOAT32) {
            		has_rgb = true;
		            rgb_offset = field.offset;
		    } else {
                rec.log(entity_path, rerun::TextLog("Only UINT32 and FLOAT32 rgb field supported"));
                continue;
            }
```